smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/HttpProtocolGeneratorUtils.java

```diff
@@ -108,6 +108,30 @@ static void generateMetadataDeserializer(GenerationContext context, SymbolRefere
         writer.write("");
     }
 
+    static void generateCollectBody(GenerationContext context) {
```

Comment:
Can you add doc strings to these?

Reply:
Added

Reply:
To be clear I meant method-level javadocs as all the other methods in this class have.